### PR TITLE
Fix concurrent push

### PR DESCRIPTION
**Pull request by NguyHa03**
The pushing step would fail if there are 2 or more repos do push at the same time to another repo, in that case we need to rebase the commit before pushing again.

### CODE REVIEW

**Comment by NguyHa03** [comment]
@dmnemec please help me review this PR

**Comment by dmnemec** [comment]
Just saw the notification. I'll check this out this weekend when I get some time. Pretty sure I'll need to write a new test for this case.

File: README.md [hunk 1]
```diff
@@ -38,6 +38,6 @@ The `API_TOKEN_GITHUB` needs to be set in the `Secrets` section of your reposito
 * destination_branch_create: [optional] A branch to be created with this commit, defaults to commiting in `destination_branch`
 * commit_message: [optional] A custom commit message for the commit. Defaults to `Update from https://github.com/${GITHUB_REPOSITORY}/commit/${GITHUB_SHA}`
 * use_rsync: [optional] Uses `rsync -avh` instead of `cp -R` to perform the base operation. Currently works as an experimental feature (due to lack of testing) but can speed up updates to large collections of files with many small changes by only syncing the changes and not copying the entire contents again. Please understand your use case before using this, and provide feedback as issues if needed.
-
+* retry_attempts: [optional] The number of retries when pushing commit failed, defaults are 10 times.
```
Comment (dmnemec) [inline README.md:41]: Minor edit to trigger checks
```suggestion
* retry_attempts: [optional] The number of retries when pushing commit failed.  Default: 10
```

**Comment by dmnemec** [comment]
Okay, checks are failing. This PR (and all others) should now run checks when you commit changes. Failure looks like it might be related to the branch names for the receiving repo, I'll have to dig into it. There's only a `main` and `dev` branch on that repo right now.

**Pull request closed without merging**